sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -207,41 +200,23 @@ export class EventHubClient {
    * @param host - Fully qualified domain name for Event Hubs. Most likely,
    * <yournamespace>.servicebus.windows.net
    * @param eventHubPath - EventHub path of the form 'my-event-hub-name'
-   * @param tokenProvider - Your token provider that implements the TokenProvider interface.
-   * @param options - The options that can be provided during client creation.
-   */
-  constructor(host: string, eventHubPath: string, tokenProvider: TokenProvider, options?: EventHubClientOptions);
-  /**
-   * @constructor
-   * @param host - Fully qualified domain name for Event Hubs. Most likely,
-   * <yournamespace>.servicebus.windows.net
-   * @param eventHubPath - EventHub path of the form 'my-event-hub-name'
-   * @param credentials - The Token credentials as implemented in the `ms-rest-nodeauth` library. It can be one of the following:
-   *  - ApplicationTokenCredentials
-   *  - UserTokenCredentials
-   *  - DeviceTokenCredentials
-   *  - MSITokenCredentials.
+   * @param tokenCredential - Your token credential that implements the TokenCredential interface.
    * @param options - The options that can be provided during client creation.
    */
   constructor(
     host: string,
     eventHubPath: string,
-    credentials: ApplicationTokenCredentials | UserTokenCredentials | DeviceTokenCredentials | MSITokenCredentials,
+    tokenCredential: SharedKeyCredential | TokenCredential,
```

Comment:
The name should be `credential`

Reply:
What do you think about userCredential or clientCredential?

Because I already have one local variable named "credential".

Reply:
You can rename the local variable to something else if you want. The suggestion from the board was to use `credential` as the argument name